confmap/expand_test.go

```diff
@@ -185,6 +185,11 @@ func TestResolverExpandStringValues(t *testing.T) {
 			input:  "test_${env:BOOL}",
 			output: "test_true",
 		},
+		{
+			name:   "Bool",
```

Comment:
The name of the test doesn't match what it is testing, could you make it:

```suggestion
			name:   "Timestamp",
```